libminifi/test/unit/SerializationTests.cpp

```diff
@@ -16,130 +16,237 @@
+  uuid_t fakeUUID;
 
-	REQUIRE( uuid_str == protocol.getPortId() );
+  uuid_parse(uuid_str.c_str(), fakeUUID);
 
+  protocol.setPortId(fakeUUID);
 
+  REQUIRE(uuid_str == protocol.getPortId());
 
 }
 
-TEST_CASE("TestSetPortIdUppercase", "[S2S2]"){
-
+TEST_CASE("TestSetPortIdUppercase", "[S2S2]") {
 
-  std::unique_ptr<minifi::Site2SitePeer> peer = std::unique_ptr<minifi::Site2SitePeer>( new minifi::Site2SitePeer(std::unique_ptr<DataStream>(new DataStream()),"fake_host",65433));
+  std::unique_ptr<minifi::Site2SitePeer> peer =
+      std::unique_ptr < minifi::Site2SitePeer
+          > (new minifi::Site2SitePeer(
+              std::unique_ptr < DataStream > (new DataStream()), "fake_host",
+              65433));
 
   minifi::Site2SiteClientProtocol protocol(std::move(peer));
 
+  std::string uuid_str = "C56A4180-65AA-42EC-A945-5FD21DEC0538";
 
-	std::string uuid_str = "C56A4180-65AA-42EC-A945-5FD21DEC0538";
-
-	uuid_t fakeUUID;
-
-	uuid_parse(uuid_str.c_str(),fakeUUID);
-
-	protocol.setPortId(fakeUUID);
+  uuid_t fakeUUID;
 
-	REQUIRE( uuid_str != protocol.getPortId() );
+  uuid_parse(uuid_str.c_str(), fakeUUID);
 
-	std::transform(uuid_str.begin(),uuid_str.end(),uuid_str.begin(),::tolower);
+  protocol.setPortId(fakeUUID);
 
-	REQUIRE( uuid_str == protocol.getPortId() );
+  REQUIRE(uuid_str != protocol.getPortId());
 
+  std::transform(uuid_str.begin(), uuid_str.end(), uuid_str.begin(), ::tolower);
 
+  REQUIRE(uuid_str == protocol.getPortId());
 
 }
 
-
-TEST_CASE("TestWriteUTF", "[MINIFI193]"){
+TEST_CASE("TestWriteUTF", "[MINIFI193]") {
 
   DataStream baseStream;
 
   Serializable ser;
 
   std::string stringOne = "helo world"; // yes, this has a typo.
   std::string verifyString;
-  ser.writeUTF(stringOne,&baseStream,false);
-
+  ser.writeUTF(stringOne, &baseStream, false);
 
-  ser.readUTF(verifyString,&baseStream,false);
+  ser.readUTF(verifyString, &baseStream, false);
 
   REQUIRE(verifyString == stringOne);
 
+}
 
+TEST_CASE("TestWriteUTF2", "[MINIFI193]") {
 
+  DataStream baseStream;
 
-}
+  Serializable ser;
 
+  std::string stringOne = "hel\xa1o world";
+  REQUIRE(11 == stringOne.length());
+  std::string verifyString;
+  ser.writeUTF(stringOne, &baseStream, false);
 
+  ser.readUTF(verifyString, &baseStream, false);
+
+  REQUIRE(verifyString == stringOne);
 
+}
 
-TEST_CASE("TestWriteUTF2", "[MINIFI193]"){
+TEST_CASE("TestWriteUTF3", "[MINIFI193]") {
 
   DataStream baseStream;
 
   Serializable ser;
 
-  std::string stringOne = "hel\xa1o world";
-  REQUIRE(11 == stringOne.length());
+  std::string stringOne = "\xe4\xbd\xa0\xe5\xa5\xbd\xe4\xb8\x96\xe7\x95\x8c";
+  REQUIRE(12 == stringOne.length());
   std::string verifyString;
-  ser.writeUTF(stringOne,&baseStream,false);
-
+  ser.writeUTF(stringOne, &baseStream, false);
 
-  ser.readUTF(verifyString,&baseStream,false);
+  ser.readUTF(verifyString, &baseStream, false);
 
   REQUIRE(verifyString == stringOne);
 
+}
 
+void sunny_path_bootstrap(SiteToSiteResponder *collector) {
```

Comment:
Either way is OK. i want to make it such that it was easy to find all tests related to Serialization.